make_mock_json.py

```diff
@@ -0,0 +1,404 @@
+        self.imagestream_server = args.image_server
+        self.imagestream_tags = args.image_tags
+
+    def comp_write(self, out_name, out_data):
+        out_path = "/".join([self.mock_dir, out_name])
+        with open(out_path, "w") as outf:
+            try:
+                json.dump(out_data, outf, indent=4)
+            except (ValueError, TypeError):
+                outf.write(json.dumps(out_data, indent=4))
+
+    def create_mock_builds_list(self):
+        kwargs = {}
+        # get build list
+        self.osbs.list_builds(**kwargs)
+        # find 'get-namespaces_osbs-stage_builds_-000.json' file and parse it into
+        # 'builds_list.json', 'builds_list_empty.json', 'builds_list_one.json'
+        all_builds = "get-namespaces_osbs-stage_builds_-000.json"
+        all_builds_path = "/".join([self.capture_dir, all_builds])
+        with open(all_builds_path, "r") as infile:
+            builds_data = json.load(infile)
+            builds_items = copy.copy(builds_data["items"])
+            builds_data["items"] = []
+            self.comp_write("builds_list_empty.json", builds_data)
+            if not builds_items:
+                return
+            builds_data["items"].append(builds_items[0])
+            self.comp_write("builds_list_one.json", builds_data)
+            if len(builds_items) < 2:
+                return
+            builds_data["items"].append(builds_items[1])
+            self.comp_write("builds_list.json", builds_data)
+        os.remove(all_builds_path)
+
+    def create_pods_list(self, build_id):
+        self.osbs.get_pod_for_build(build_id)
+        pods_pre = "get-namespaces_osbs-stage_pods_?labelSelector=openshift.io%2Fbuild.name%3D"
+        for i in range(0, 4):
+            try:
+                pods_fname = pods_pre + build_id + "-00{}.json".format(i)
+                pods_inpath = "/".join([self.capture_dir, pods_fname])
+                os.stat(pods_inpath)
+                break
+            except OSError:
+                continue
+
+        with open(pods_inpath, "r") as infile:
+            pods_data = json.load(infile)
+            image = "buildroot:latest"
+            pods_items = pods_data["items"] or []
+            for pod in pods_items:
+                pod_containers = pod["status"]["containerStatuses"] or []
+                for container in pod_containers:
+                    container["imageID"] = "docker-pullable://" + image
+                    container["image"] = image
+            self.comp_write("pods.json", pods_data)
+        os.remove(pods_inpath)
+
+    def create_mock_get_user(self):
+        self.osbs.get_user()
+        user_list = "get-users_~_-000.json"
+        user_list_path = "/".join([self.capture_dir, user_list])
+        with open(user_list_path, "r") as infile:
+            user_data = json.load(infile)
+            user_data["groups"] = []
+            user_data["identities"] = None
+            if user_data["fullName"]:
```

Comment:
did you mean `if "fullName" in user_data:`?

Reply:
Probably.

Fixed, thanks.